pipeline/direct/direct.py

```diff
@@ -328,17 +307,21 @@ def get_hazard(haz_type, country_iso3alpha, scenario, ref_year):
     if haz_type == 'tropical_cyclone':
         if scenario == 'None' and ref_year == 'historical':
             return client.get_hazard(
-                haz_type, properties={
+                'tropical_cyclone',
```

Comment:
(Cosmetic) Any reason why we use the hardcoded one here?
All the other statements on the bottom use `haz_type`

Reply:
Not really sure! I would say it's worth hard-coding because 'tropical_cyclone' is a parameter sent to the CLIMADA API and there might one day be a situation where our NCCS haz_type is not tropical_cyclone?

It's a bit of a thin justification though 😁